service/test/test_testcase_api.py

```diff
@@ -56,6 +56,9 @@ def test_ibpsa_boptest_testcase():
     response = requests.get(f"{host}/name/{testid}")
     check.equal(response.status_code, 200)
 
+    response = requests.post(f"{host}/advance/{testid}")
```

Comment:
A lot of API tests in this file, but there was not actaully a test for advance, which is a useful assertion when working on timeout issues.